stdlib/http/src/main/ballerina/http/Module.md

```diff
@@ -51,21 +57,24 @@ There are also two types of services for WebSocket: `WebSocketService` and `WebS
     }
 }
 ```
-Here `upgradeService` is a `WebSocketService`.
+Here `upgradeService` is a server callback service.
 
-**onOpen resource**: As soon as the WebSocket handshake is completed and the connection is established, the `onOpen` resource is dispatched. This resource is only available in the `WebSocketService`.
+**onOpen resource**: As soon as the WebSocket handshake is completed and the connection is established, the `onOpen` 
+resource is dispatched. This resource is only available in the server service.
 
 **onText resource**: Received Text messages are dispatched to this resource.
 
 **onBinary resource**: Received Binary messages are dispatched to this resource.
 
-**onPing and onPong resources**: Received ping and pong messages are dispatched to these resources respectively as a `blob`
+**onPing and onPong resources**: Received ping and pong messages are dispatched to these resources respectively.
 
-**onIdleTimeout**: This resource is dispatched when idle timeout is reached. idleTimeout has to be configured by the user in WebSocket service configuration.
+**onIdleTimeout**: This resource is dispatched when idle timeout is reached. idleTimeout has to be configured by the 
+user in WebSocket service or client configuration.
 
 **onClose**: This resource is dispatched when a close message is received.
```

Comment:
It is not clear what a close message is.

Reply:
Fixed. Please check.